<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.adv.qa.selenium.tests.currency.phase3.AD01008_IM_Audit_ReportsOracleTest" tests="1" time="65.698" ignored="0" failures="0" timestamp="19 May 2017 10:49:14 GMT" skipped="0" hostname="ChetnaM-PC" errors="1">
  <testcase name="verify" time="65.698" classname="com.adv.qa.selenium.tests.currency.phase3.AD01008_IM_Audit_ReportsOracleTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.adv.qa.selenium.helpers.DatabaseQuery_Oracle.getStatProcessOracle(DatabaseQuery_Oracle.java:44)
	at com.adv.qa.selenium.framework.pageObjects.currency.CurrencyPage.getProcessDetailsOracle(CurrencyPage.java:9154)
	at com.adv.qa.selenium.tests.currency.phase3.AD01008_IM_Audit_ReportsOracleTest.submitProcess(AD01008_IM_Audit_ReportsOracleTest.java:75)
	at com.adv.qa.selenium.tests.currency.phase3.AD01008_IM_Audit_ReportsOracleTest.verify(AD01008_IM_Audit_ReportsOracleTest.java:54)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </error>
  </testcase> <!-- verify -->
</testsuite> <!-- com.adv.qa.selenium.tests.currency.phase3.AD01008_IM_Audit_ReportsOracleTest -->
